### ISSUE TITLE
bug: Ion-menu - Cannot read properties of undefined (reading 'offsetWidth')

### ISSUE THREAD

**Issue by Yura13** [ionic-team/ionic-framework#24994]
### Prerequisites

- [X] I have read the [Contributing Guidelines](https://github.com/ionic-team/ionic-framework/blob/main/.github/CONTRIBUTING.md#creating-an-issue).
- [X] I agree to follow the [Code of Conduct](https://ionicframework.com/code-of-conduct).
- [X] I have searched for [existing issues](https://github.com/ionic-team/ionic-framework/issues) that already report this problem, without success.

### Ionic Framework Version

- [ ] v4.x
- [X] v5.x
- [X] v6.x

### Current Behavior

I got an error **Cannot read properties of undefined (reading 'offsetWidth')** in the console on first trying to open ion-menu.
Ion-menu opens only after the refresh page.
![image](https://user-images.githubusercontent.com/17030431/159902134-f41d4d3d-690c-4a3a-abd9-815812333649.png)

### Expected Behavior

Ion-menu should open always without any error.

### Steps to Reproduce

It happens only right after building/running the project.

### Ionic Info

Ionic:

   Ionic CLI                     : 6.18.1 (C:\Users\Xiaomi\AppData\Roaming\nvm\v16.13.2\node_modules\@ionic\cli)
   Ionic Framework               : @ionic/angular 6.0.13
   @angular-devkit/build-angular : 13.3.0
   @angular-devkit/schematics    : 13.3.0
   @angular/cli                  : 13.3.0
   @ionic/angular-toolkit        : 5.0.3

Cordova:

   Cordova CLI       : 10.0.0 (cordova-lib@10.1.0)
   Cordova Platforms : android 9.1.0
   Cordova Plugins   : cordova-plugin-ionic-keyboard 2.2.0, cordova-plugin-ionic-webview 5.0.0, cordova-plugin-ionic 5.5.1, (and 30 other plugins)

Utility:

   cordova-res : not installed globally
   native-run  : 1.5.0

System:

   Android SDK Tools : 26.1.1 (C:\Android\android-sdk)
   NodeJS            : v16.13.2 (C:\Program Files\nodejs\node.exe)
   npm               : 8.1.2
   OS                : Windows 10

**Comment by liamdebeasi** [comment]
Thanks for the issue. I am going to close this as a duplicate of https://github.com/ionic-team/ionic-framework/issues/24907.
